db/migrate/20180905101200_reindex_vacancies_after_changing_max_salary_type.rb

```diff
@@ -1,5 +0,0 @@
-class ReindexVacanciesAfterChangingMaxSalaryType < ActiveRecord::Migration[5.2]
```

Comment:
Should we be removing migrations?

Reply:
It's a fake migration which does not touch the database, but just runs a task. But your question still stands.
https://github.com/DFE-Digital/teacher-vacancy-service/commit/825e85c2ecffe8596734f7b09f6768bae7b78827

Reply:
Ok cool, I just wasn't sure how migrations work in ActiveRecord

Reply:
Given the way Rails tracks migrations, and the fact that they can be fiddly and difficult to maintain, a better solution would be to leave the migration in place but this:

```ruby
  def change
     #noop
     # <brief explaination>
  end
```

Reply:
Otherwise, migration versioning numbers may well get out of sync which can cause problems maintaining `schema.rb` and make it difficult to reconcile versions later.